### DISCUSSION TITLE
Making request authorization simple.

### DISCUSSION THREAD

**Discussion by 8ctavio** [axios/axios#4729] [Ideas]
A feature I'd like to see in Axios is an easy and automated way to handle some complex authorization types like digest auth. For reference, Postman has a very simple way for testing requests with a variety of auth models; but other than Postman (and maybe other similar apps for testing requests) I haven't seen any major, popular and active package to simplify authorization for http requests.

**Comment by jasonsaayman** [comment]
Please make a pr, we are open to suggestions 😄
